code/web/src/modules/survey/Survey/Item.js

```diff
@@ -27,22 +28,25 @@ class Item extends PureComponent {
   }
 
   handleItemClick = () => {
+    console.log(this.state.selected)
     this.setState({ selected: !this.state.selected });
-    updateStoredItems(this.props.item, this.state.selected)
+    updateSelectedItems(this.props.item, this.state.selected)
   }
 
   render() {
-    <img
-      src={this.props.item.image}
-      style={{
-        height: '5em',
-        width: '5em',
-        borderStyle: 'solid',
-        borderWidth: '0.5em',
-        borderColor: this.state.selected ? primaryAccent : primary
-      }}
-      onClick={this.updateStoredItems}>
-    </img>
+    return (
+      <img
+        src={routeImage + this.props.item.image}
+        style={{
+          height: '5em',
+          width: '5em',
+          borderStyle: 'solid',
+          borderWidth: '0.5em',
+          borderColor: this.state.selected ? primaryAccent : primary
```

Comment:
When looking at the mockup do you think that the colors would be easily visible?

Reply:
Yep! Goo thought